docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -683,7 +689,7 @@
                 </tr>
 
                 <tr>
-                    <th colspan="3" class="section" id="streams"><a href="../container/streams.html">Systems (input and output streams)</a></th>
+                    <th colspan="3" class="section" id="systems"><a href="../container/streams.html">Systems (input and output streams)</a></th>
```

Comment:
Seems to me it is better to change the titles to Streams instead of the id to "systems". Anyways, we are talking about "Streams" here and the link is also to streams.html.